setup.py

```diff
@@ -48,7 +48,9 @@ def run(self):
 
                         f.flush()
 
-        dependencies = os.path.abspath("imagej/jars/cellprofiler-java-dependencies-classpath.txt")
+        dependencies = os.path.abspath(os.path.join(
```

Comment:
I needed to join self.build_lib to "imagej/jars/cellprofiler-java-dependencies-classpath.txt", so I broke it up into separate directory names. But os.join(self.build_lib, 'imagej...') probably works too.